apps/fabric-website/src/pages/Styles/FluentThemePage/docs/web/FluentThemeOverview.md

```diff
@@ -12,6 +12,10 @@ We’re updating Fabric to include a new set of Web Fluent design styles, which
 
 We're tracking updates to the Fabric React components in [this GitHub project](https://github.com/OfficeDev/office-ui-fabric-react/projects/23). Fluent styles for components will continue to be added to the `@uifabric/fluent-theme` package for Fabric React 6. They will become the default styles for Fabric React components in v7, which we aim to release by end of May 2019.
 
+You can view the new Fluent web controls in the implementation section on the control pages. Click the tab dropdown to toggle between our current and future Fluent controls.
+
+![screenshot of theme dropdown on control examples](https://static2.sharepointonline.com/files/fabric/fabric-website/images/control-fluent-dropdown.png)
```

Comment:
Should we check these screenshots in the `fabric-website-resources`, @ecraig12345 & @jordandrako ? Don't know if this has been the practice all along maybe...

Reply:
We've been uploading images/etc to a CDN separately. I think that's still the right approach because the images don't change very often and don't need to be re-deployed every build.

Reply:
By "uploading" I mean checking them in to an internal repo, and I'm guessing the contents of that are pushed to the CDN by some build definition.

Reply:
@ecraig12345 Is correct, images should be in the fabric-cdn repo.